components/SideNavigationBar/index.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import SideNavigationBar from './SideNavigationBar';
+import LogOutButton from './LogOutButton';
+
+
+export default function admin() {
+    const NavigationBarData1 = [
+        {
+          title: "Home",
+          image: "/Images/SideNavigationBar/home.png",
+          alt: "Home Image",
+          button: true
+        },
+        {
+          title: "Dashboard",
+          image: "/Images/SideNavigationBar/dashboard.png",
+          alt: "Dashboard Image",
+          button: true
+        },
+        {
+          title: "Projects",
+          image: "/Images/SideNavigationBar/project.png",
+          alt: "Projects Image",
+          button: true
+        },
+        {
+          title: "Tasks",
+          image: "/Images/SideNavigationBar/task.png",
+          alt: "Task Image",
+          button: true
+        },
+        {
+          title: "Reporting",
+          image: "/Images/SideNavigationBar/report.png",
+          alt: "Reporting Image",
+          button: true
+        },
+        {
+          title: "Users",
+          image: "/Images/SideNavigationBar/user.png",
+          alt: "Users Image",
+          button: true
+        },
+      // Add more card data objects as needed
+    ];
+
+    const NavigationBarData2 = [
+      {
+        title: "Support",
+        image: "/Images/SideNavigationBar/support.png",
+        alt: "Support Image",
+        button: true
+      },
+      {
+        title: "Settings",
+        image: "/Images/SideNavigationBar/settings.png",
+        alt: "Settings Image",
+        button: true
+      }
+    // Add more card data objects as needed
+  ];
+
+  return (
+
+    <div>
+      {/* <Disclosure as="nav">
```

Comment:
Delete commented out code unless planning to uncomment in the future.

Reply:
Fix the issue